### DISCUSSION TITLE
Images similar to Adobe Firefly Text Effects in Fooocus?

### DISCUSSION THREAD

**Discussion by digitrip** [lllyasviel/Fooocus#715] [Q&A]
Hello everyone,

I have just started to experiment with this wonderful creative tool, Fooocus sdxl, (no previous experience in ai art), and I am trying unsuccessfully to generate images similar to [Adobe Firefly Text Effects](https://www.google.com/search?q=firefly+text+effects&tbm=isch).

I have done a search for similar discussions and the only solution I found is [sd-text-effects](https://github.com/ihsavru/sd-text-effects), but it is for Stable Diffusion Web UI by AUTOMATIC1111.

Is it possible in Fooocus to generate results similar to Adobe Firefly Text Effects?
And if so, what is the most effective way?

Thank you

**Comment by barepixels** [comment]
Short answer: Is it NOT possible in Fooocus.  For now use A1111

**Reply by digitrip** [reply]
Thank you

**Comment by lllyasviel** [comment]
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/4505eef1-d345-4325-9c76-41e344fa8e11)

need cherry pick and special prompt and bg removal

**Reply by digitrip** [reply]
Thank you very much for the detailed answer, it was really helpful.
Experimenting with Fooocus is a pure pleasure.
Are there any plans for something like [Adobe Firefly Text Effects](https://www.google.com/search?q=firefly+text+effects&tbm=isch) or [sd-text-effects](https://github.com/ihsavru/sd-text-effects) for Fooocus in the future?
